packages/lesswrong/components/common/CommentsListCondensed.tsx

```diff
@@ -36,7 +49,15 @@ const CommentsListCondensed = ({label, terms, initialLimit, itemsPerPage, showTo
 
   const showLoadMore = !loading && (count === undefined || totalCount === undefined || count < totalCount)
   return <>
-    <SectionTitle title={label} className={classes.subheader} />
+    <SectionTitle title={label} className={classes.subheader} >
+      {currentUser?.isReviewed && shortformButton && !currentUser.allCommentingDisabled && <div onClick={toggleShortformFeed}>
+        <SectionButton>
+          <AddBoxIcon />
+          New Shortform Post
```

Comment:
Maybe this should just say "New" now?

Reply:
I'm chill with it being overly explicit

Reply:
or "New shortform" to help differentiate from "New post" 🤔